### PR TITLE
Refs #26393 - import boot time information

### PR DESCRIPTION
**Pull request by ares**
needs https://github.com/theforeman/foreman/pull/6843

### CODE REVIEW

File: app/models/katello/rhsm_fact_parser.rb [hunk 1]
```diff
@@ -72,6 +72,10 @@ def environment
     def ipmi_interface
     end
 
+    def boot_time
+      facts['proc_stat::btime']&.to_i
```
Comment (ezr-ondrej) [inline app/models/katello/rhsm_fact_parser.rb]: Shouldn't this be `proc_stat.btime`?

**Comment by ares** [comment]
of course, thanks for pointing that out

**Comment by ares** [comment]
@ezr-ondrej I belive you tested this recently, perhaps you could ack :-)

**Comment by jlsherrill** [comment]
[test katello]

**Comment by jlsherrill** [comment]
have never seen this test failure before, but let's [test katello]

**Comment by ares** [comment]
:green_apple:

**Pull request merged**